src/UI/Implementation/Render/ilJavaScriptBinding.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/* Copyright (c) 2016 Richard Klees <richard.klees@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Implementation\Render;
+
+/**
+ * Wraps global ilTemplate to provide JavaScriptBinding. 
+ */
+class ilJavaScriptBinding implements JavaScriptBinding {
+	/**
+	 * @var \ilTemplate
+	 */
+	private	$global_tpl;
+
+	/**
+	 * @var	integer
+	 */
+	private $id_count = 0;
```

Comment:
Is it possible, that we have two binders working in parallel or subsequently in future for the same screen? E.g. I think of async requests to get a further Modal or similar. If so, should we take better care to randomize ids somehow to make collision extremely unprobable?